### PR TITLE
Create IFrameRoom.

### PR DESCRIPTION
**Pull request by seangibat**
- Create the IFrameRoom room type
- Add a room to the map pointed at the DJ Friend

### CODE REVIEW

File: app/client/jsx/Welcome.jsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Welcome extends Component {
         return (
             <div className="vestibule">
                 <h1>Welcome have fun</h1>
-                <input style={text_entry} className={name_opacity} type="text" placeholder="Name" name="name" minLength="1" onChange={this.handleDisplayNameChange.bind(this)}/><br/>
+                <input style={text_entry} autoComplete="off" className={name_opacity} type="text" placeholder="Name" name="name" minLength="1" onChange={this.handleDisplayNameChange.bind(this)}/><br/>
```
Comment (seangibat) [inline app/client/jsx/Welcome.jsx:57]: I added this because it the autocomplete was slowing down my dev process on reload.

**Comment by seangibat** [comment]
`I chose to use the names `IFrameRoom` and `iframeRoom`, which is not totally coherent, but I had a distaste for `iFrameRoom`.

**Comment by morganecf** [comment]
If you pull latest changes there'll be bundle.js in the gitignore so that you don't have to commit those!

**Comment by seangibat** [comment]
Cool, ty. I went ahead and deleted the bundles.

**Comment by seangibat** [comment]
Merge it!!!!!!!!!!!

**Pull request merged**